### ISSUE TITLE
incompatible with tensorflow 2

### ISSUE THREAD

**Issue by tfriedel** [Yoctol/serving-utils#53]
``` py
from serving_utils import Client, PredictInput
```

--> 
```
-------------------------------------------------------------------
AttributeError                    Traceback (most recent call last)
<ipython-input-1-79296e6888bc> in <module>
----> 1 from serving_utils import Client, PredictInput

/usr/local/lib/python3.6/dist-packages/serving_utils/__init__.py in <module>
      1 from .client import Client, PredictInput
----> 2 from .saver import Saver
      3 from .loader import Loader

/usr/local/lib/python3.6/dist-packages/serving_utils/saver.py in <module>
      6 
      7 
----> 8 class Saver:
      9 
     10     def __init__(

/usr/local/lib/python3.6/dist-packages/serving_utils/saver.py in Saver()
     13             output_dir: str,
     14             signature_def_map: Dict[str, tf.saved_model.signature_def_utils.predict_signature_def],
---> 15             freeze: bool = True,
     16         ) -> None:
     17         self.session = session

AttributeError: module 'tensorflow' has no attribute 'Session'
```

**Comment by tfriedel** [comment]
changing ```py import tensorflow as tf```` in saver.py to
```py import tensorflow.compat.v1 as tf``` fixes it.
I don't know if it's sufficient. There's also this tool that converts code to make it compatible with tf2.

**Comment by stegben** [comment]
fixed via #54
